mod_reforged/hooks/config/tip_of_the_day.nut

```diff
@@ -19,7 +19,7 @@ for (local index = (::Const.TipOfTheDay.len() - 1); index >= 0; index--)
 }
 
 ::Const.TipOfTheDay.extend([
-    "You can interact with towns while on a caravan mission.",
+    "You can enter towns while on a caravan mission; even if they are 2 tiles away.",
```

Comment:
The semi-colon is wrong here. You can simply remove it and the sentence would be ok.